### ISSUE TITLE
TYPO3 11 Compatibility

### ISSUE THREAD

**Issue by andyhausmann** [in2code-de/femanager#332]
Are there any plans regarding TYPO3 11 compatibility?

**Comment by sbusemann** [comment]
at lastest to the LTS release. Sponsoring can make this faster.

**Comment by mavolkmer** [comment]
Hi,
any news on that?
I'm not sure about the current state of TYPO3 v11 migration for ext:femanager; saw that first preparations had been pushed into a dedicated branch.
To quickly check some stuff in a v11 project I've made some changes derived from the `feature/V11` branch. It´s quick an dirty but at least one can register new users with TYPO3-v11 and ext:femanager with that:
https://github.com/mavolkmer/femanager/tree/typo3-11
If it's useful I could open a PR with my changes - I could also provide some further help if needed.

**Comment by sbusemann** [comment]
any contribution is welcome, but you need to take care about the quality requirements  - see https://github.com/in2code-de/femanager#your-contribution

Biggest blocker: At the moment the Behat Tests do not work. Also our team is working on Powermail. After this is finished we continue here. For the Future we will introduce an Early Access Programm for Sponsors and will provide updates much earlier.

**Comment by liveman1** [comment]
Hi, do you plan to release a v11 compatible version in the near future?
It would be great to get a rough idea.
Thanks a lot!

**Comment by sbusemann** [comment]
yes it is planned in Q2 - users of https://www.in2code.de/en/agency/typo3-extensions/early-access-program/ will get it first.

**Comment by sbusemann** [comment]
The version is available in the EAP. The public release will be within the next two weeks.

**Comment by bahneklammerauf** [comment]
Hi, will the public release be available soon?

**Comment by mediaessenz** [comment]
Is there any news regarding the release?

You, @sbusemann wrote that it will be published in the next two weeks.

That was four weeks ago ...

**Comment by pietvb93** [comment]
Hi, 
do you still plan to release a new version anytime soon or are there any issues?
Please recommend if we should focus on alternative extensions instead.

Many thanks, Piet

**Comment by mediaessenz** [comment]
@ulrike-cosmoblonde What do you mean with "@dev" version?

**Comment by ulrike-cosmoblonde** [comment]
@mediaessenz When you are using composer, you can simply define a version with your require like this
`composer require in2code/femanager:@dev`
